src/App.css

```diff
@@ -1,4 +1,4 @@
-.App {
+/* .App {
```

Comment:
- [ ] Please delete the `App.css` file since its content are commented and, therefore, not in use 👍